### ISSUE TITLE
Unable to import entity with partition key 'xx' and row key 'yy'

### ISSUE THREAD

**Issue by aditheppru** [microsoft/AzureStorageExplorer#5395]
After upgrading Microsoft Azure Storage Explorer to version 1.22.1, when I imported a csv file, I got a dialog error "Unable to import entity with partition key 'xx' and row key 'yy'". I understand that the row with the partition key and row key from the csv file already exists in the target storage table. However, before I upgraded the Explorer to version 1.22.1, I had a csv file containing duplicate and new data and I could import the file. Please bring this feature back or the Explorer should provide users with an option to choose if they want to import & replace or just want to import new data.

Regards,
Adithep Pruekpitakpong

**Comment by craxal** [comment]
@aditheppru Thanks for letting us know. This is a known issue that we're working on. If you are blocked by this issue, you can revert back to [1.21.3](https://github.com/microsoft/AzureStorageExplorer/releases/tag/v1.21.3) until a fix is available.

Duplicate of #5369
